string name
float64 value
---
bool ok
